src/parser.js

```diff
@@ -77,56 +130,29 @@ export function containIncludeCommand(commands = []) {
  */
 export function parseVariablesFromLabel(kvMap, label) {
     const kv = Object.assign({}, kvMap);
-    const beginEx = "^.*";
-    const endEx = ".*$";
-    const sepEx = ",?";
-    const kvsepEx = "[:=]";
-    const spacesEx = "\\s*";
-    const quotesEx = "[\"']";
-
-    Object.keys(kv).forEach(key => {
-        let keyEx = "(" + key + ")";
-        let valEx = "([-\\w\\s]*)";
-        if (key === "marker") {
-            keyEx = "(import|include)";
-            valEx = "(([-\\w\\s]*,?)*)";
+
+    let match = "";
+    while ((match = expressionRegExp.exec(label))) {
+        let key = match[1];
+        if (["include", "import"].includes(key)) {
```

Comment:
> Shouldn't this be handled by babel instead of by us? 

No. Babel only transpile syntax by default.

- https://babeljs.io/docs/usage/polyfill/
- https://github.com/babel/babel-preset-env#usebuiltins
  - Need to add `import "babel-polyfill";`

I think  that this PR fix a bug - it is a patch version update.
Dropping Node.js 4.x make breaking change - it is a major version update.